app/shop.py

```diff
@@ -0,0 +1,32 @@
+from math import sqrt
+
+from app.customer import Customer
+
+
+class Shop:
+    def __init__(self, name: str, location: list, products: dict) -> None:
+        self.name = name
+        self.location = location
+        self.products = products
+
+    def calculate_distance(self, customer: Customer) -> float:
+        shop_x, shop_y = self.location
+        customer_x, customer_y = customer.location
+        distance = sqrt(
+            ((shop_x - customer_x) ** 2)
+            + ((shop_y - customer_y) ** 2))
+        return distance

```

Comment:
you can use `math.dist`

Reply:
i know but i decided to keep own realization